src/custom/hooks/useRemainingAllowanceToApprove.ts

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react'
```

Comment:
originally coded this because I thought we were approving only investing amount but it is no longer in use (as we use max unit256) - figured I'd leave it as it's useful for if we switch to this and/or can use in other parts of the app if we want to do exact approvals (it calcs current approved vs amount requested)

happy to remove.